ui/pages/confirm-decrypt-message/confirm-decrypt-message.component.js

```diff
@@ -19,6 +19,7 @@ import { formatCurrency } from '../../helpers/utils/confirm-tx.util';
 import { getValueFromWeiHex } from '../../../shared/modules/conversion.utils';
 import { COPY_OPTIONS } from '../../../shared/constants/copy';
 
+// As of this PR; This component is is only used in a test file
```

Comment:
It's worth considering creating a ticket or multiple tickets to track the deletion of dead code. This way the work can be triaged and allocated to be actioned by a team.